package.json

```diff
@@ -0,0 +1,99 @@
+{
+  "name": "automation-webapi",
+  "version": "1.0.0",
+  "description": "",
+  "main": "src/app.js",
+  "scripts": {
+    "build": "export BABEL_ENV=production && gulp clean && babel src -d build/src && gulp && export BABEL_ENV=development",
+    "dev": "nodemon  src/app.js",
+    "prestart": "npm install --only=production",
+    "start": "pm2 start process.json --env production",
+    "stop": "pm2 stop process.json",
+    "lint": "eslint --fix ./src",
+    "testTmp": "nodemon  src/modules/util/userstatusCheck.test.js"
+  },
+  "private": true,
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@ccwebprd/node-crate": "^2.0.9",
+    "@ccwebprd/pgsql-parser": "^0.5.10",
+    "adm-zip": "^0.4.13",
+    "babel-runtime": "^6.26.0",
+    "backo": "^1.1.0",
+    "bcryptjs": "^2.4.3",
+    "biguint-format": "^1.0.1",
+    "bluebird": "^3.5.1",
+    "bunyan": "^1.8.12",
+    "bunyan-rotating-file-stream": "^1.6.3",
+    "chance": "^1.0.13",
+    "csvtojson": "^2.0.8",
+    "debug": "^4.1.1",
+    "exceljs": "^0.4.14",
+    "file-timestamp-stream": "^2.2.0",
+    "flake-idgen": "^1.1.2",
+    "fs-extra": "^5.0.0",
+    "glob": "^7.1.6",
+    "html-entities": "^1.2.1",
+    "iconv-lite": "^0.5.1",
+    "ioredis": "^4.2.0",
+    "joi": "^13.1.0",
+    "json-bigint": "^0.3.0",
+    "key-del": "^1.3.0",
+    "knex": "^0.14.2",
+    "koa": "^2.4.1",
+    "koa-body": "^4.0.0",
+    "koa-bodyparser": "2.x",
+    "koa-convert": "^1.2.0",
+    "koa-error": "^2.1.0",
+    "koa-router": "^7.3.0",
+    "lodash": "^4.17.15",
+    "lru-cache": "^5.1.1",
+    "md5-file": "^5.0.0",
+    "moment": "^2.20.1",
+    "moment-timezone": "^0.5.33",
+    "mysql2": "^1.5.1",
+    "nedb": "^1.8.0",
+    "node-forge": "^0.7.5",
+    "node-rdkafka": "^2.7.4",
```

Comment:
where is rdkafka used?

Reply:
used for common message-center module and is not used currently.